src/lib/containers/access_requirement_list/AccessRequirementList.tsx

```diff
@@ -217,6 +236,24 @@ export default function AccessRequirementList({
         return undefined
     }
   }
+
+  const requestDataStepCallback = (props:requestDataStepCallbackProps) => {
+    const {managedACTAccessRequirement, step, researchProjectId, formSubmitRequestObject} = props
+    if (managedACTAccessRequirement) {
+      // to identify which managedACTAccessRequirement we are looking at in step 1 & 2 forms
+      setAccessRequirementId(String(managedACTAccessRequirement.id))
```

Comment:
This seems unnecessary if you're passing around the entire managedACTAccessRequirement.  Am I missing something?

Reply:
I did this because managedACTAccessRequirement.id is a number type and all the APIs ask for a string type. So I create an extra variable just to avoid type conversion in several places. But for clarity's sake, I removed the redundant variable.